server/api/users.js

```diff
@@ -924,87 +848,14 @@ internals.applyRoutes = function (server, next) {
     handler: function (request, reply) {
 
       const user = request.pre.user;
-
-      if (user.roles.analyst) {
-        return reply(user);
+      if (request.params.role in Config.get('/role')){
+        reply(user);
```

Comment:
remove this block of code? what is this check for? we obviously never pass this check as config.get returns an array of objects.(and not a dict like object)